src/app/modules/admin-shared/confirm-dialog/confirm-dialog.component.ts

```diff
@@ -25,6 +27,11 @@ export class ConfirmDialogComponent implements OnInit {
     this.dialogRef.close(true);
   }
 
+  toCheckStatus(){
```

Comment:
these should happen in parent components

Reply:
yes, Removed